src/plugin/relativeTime/index.js

```diff
@@ -17,21 +48,23 @@ export default (o, c, d) => {
     y: 'a year',
     yy: '%d years'
   }
+  if (o.strict) {
+    U.x(d.en.relativeTime, {
```

Comment:
can we just do 
```
d.en.relativeTime = {
  // and copy all the relevant keys
}
```

Reply:
I don't think it's a good pattern personally, but you'd like to do this, that's fine to me.

Reply:
Or, do we really have to update this? Why not just leave it to our user themself?

Reply:
Would it be much easier to use for users?